test/elixir_search_extractor_web/features/authentication_page/login_page_test.exs

```diff
@@ -0,0 +1,15 @@
+defmodule ElixirSearchExtractorWeb.AuthenticationPage.LoginPageTest do
+  use ElixirSearchExtractorWeb.FeatureCase
```

Comment:
let's add `async: true` here and in other feature tests too 👍

Reply:
Added in [bd28d6f](https://github.com/rafayet-monon/elixir_search_extractor/pull/22/commits/bd28d6feca107003370bda0276378f80c3e5a6ed)